README.md

```diff
@@ -93,6 +93,7 @@ Jump to
 [back to top](#readme)
 
 - [GEO Alarm](https://github.com/foxanna/SimpleLocationAlarm): Alarm based your location [[Android](https://play.google.com/store/apps/details?id=simplelocationalarm.droid), WP]
+- [xUber](https://github.com/CrossGeeks/xUber): Uber Clone using Xamarin Forms
```

Comment:
Could you please make some changes:
- Simple app description - `Uber Clone`
- Additional info -  `[ Forms: iOS, Android] (Features: Maps, Location)`
- Move to `Clones` section.

Reply:
yes, sure
I've fixed it
Please review it again